tasks/ansible.py

```diff
@@ -20,9 +20,15 @@ def _run(self):
 class AnsiblePlaybook(PopenTask):
     def __init__(self, playbook=None, extra_vars=None,
                  verbosity=None, **kwargs):
+        if self.extra_vars is None:
+            extra_vars = {}
+        # use Python 3 as default Python interpreter for Ansible
+        extra_vars.setdefault(
```

Comment:
This works only for `build` tasks, they are the only one calling this class, for the rest I think the best is to add that setting to the files:
- `/templates/ad.vars.yml`
- `/templates/run_pytest.vars.yml`

However, I tested it and hit the following error related to selinux:
```
2019-04-30 21:38:05,364    DEBUG  fatal: [controller]: FAILED! => {"changed": false, "checksum": "295e3e77dde23ab410afa683ad7589e9070387c6", "cur_context": ["system_u", "object_r", "nfs_t", "s0"], "gid": 0, "group": "root", "input_was": ["system_u", "object_r", "default_t", "s0"], "mode": "0744", "msg": "invalid selinux context: [Errno 95] Operation not supported", "new_context": ["system_u", "object_r", "default_t", "s0"], "owner": "root", "path": "/vagrant/.ansible_tmpusb2b4bgipa-test-config.yaml", "secontext": "system_u:object_r:nfs_t:s0", "size": 607, "state": "file", "uid": 0}
2019-04-30 21:38:05,375    DEBUG  
2019-04-30 21:38:05,375    DEBUG  msg: invalid selinux context: [Errno 95] Operation not supported
```
[Complete log here](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/195edcce-6ba9-11e9-9757-5254002f0d7e/runner.log.gz).